src/ert/_c_wrappers/enkf/queue_config.py

```diff
@@ -14,6 +15,22 @@ class QueueConfig:
         default_factory=dict
     )
 
+    @classmethod
+    def from_dict(cls, config_dict):
+        queue_system = config_dict.get("QUEUE_SYSTEM", "LOCAL")
+        queue_system = QueueDriverEnum.from_string(f"{queue_system}_DRIVER")
+        job_script = config_dict.get("JOB_SCRIPT", shutil.which("job_dispatch.py"))
```

Comment:
is there any way to move the `shutil.which` call further up the stack, and pass the result, to keep the config code "pure"?

Reply:
There are multiple ways we could move the call to `shutil.which` further up the stack but if you look at ln 10 in this file you will see that I am not really adding the `shutil` check with these changes. 

I think it can live here for now and later decide if we really want to move it away.